### PR TITLE
 feat(docs): add Eva section

### PR DESCRIPTION
**Pull request by yggg**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [x] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:

### CODE REVIEW

File: docs/app/@theme/components/eva/eva.component.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <h2 class="h1 heading">
+    <span class="pre-heading">Based on</span>
+    Eva Design System
+  </h2>
+  <p class="text">Construct consistent & scalable interface using basic components following Eva Guidelines and it will always have stunning design. </p>
+  <a class="learn-more" nbButton size="giant">Learn more about Eva</a>
+</div>
+<div class="images">
+  <img class="theme-colors"
+       src="/assets/img/theme-colors.png"
```
Comment (nnixaa) [inline docs/app/@theme/components/eva/eva.component.html]: Seems that image srcs are not correct

File: docs/app/@theme/components/eva/eva.component.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<div class="description">
+  <h2 class="h1 heading">
+    <span class="pre-heading">Based on</span>
+    Eva Design System
+  </h2>
+  <p class="text">Construct consistent & scalable interface using basic components following Eva Guidelines and it will always have stunning design. </p>
+  <a class="learn-more" nbButton size="giant">Learn more about Eva</a>
```
Comment (nnixaa) [inline docs/app/@theme/components/eva/eva.component.html]: href="https://eva.design"

File: docs/app/@theme/components/eva/eva.component.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<div class="description">
+  <h2 class="h1 heading">
+    <span class="pre-heading">Based on</span>
+    Eva Design System
+  </h2>
+  <p class="text">Construct consistent & scalable interface using basic components following Eva Guidelines and it will always have stunning design. </p>
```
Comment (nnixaa) [inline docs/app/@theme/components/eva/eva.component.html]:
```suggestion
  <p class="text">Construct stunning & consistent interfaces using atomic components by following Eva Design System specifications.</p>
```

File: docs/app/@theme/components/eva/eva.component.html (hunk 1 above)
Comment (nnixaa) [reply docs/app/@theme/components/eva/eva.component.html]: image

File: docs/app/@theme/components/eva/eva.component.html (hunk 2 above)
Comment (yggg) [reply docs/app/@theme/components/eva/eva.component.html]: Fixed in b4f17d5

File: docs/app/@theme/components/eva/eva.component.html (hunk 1 above)
Comment (yggg) [reply docs/app/@theme/components/eva/eva.component.html]: Fixed in 9f451b2

**Pull request merged**